src/ControlledBubbleMenu.tsx

```diff
@@ -9,13 +9,14 @@ import {
 import { isNodeSelection, posToDOMRect, type Editor } from "@tiptap/core";
 import { useCallback } from "react";
 import { makeStyles } from "tss-react/mui";
+import type { Except } from "type-fest";
 import { getUtilityClasses } from "./styles";
 
 export type ControlledBubbleMenuClasses = ReturnType<
   typeof useStyles
 >["classes"];
 
-export type ControlledBubbleMenuProps = {
+export type ControlledBubbleMenuProps = Except<PopperProps, "children"> & {
```

Comment:
In this case, any new props from `PopperProps` that would now be included as part of `ControlledBubbleMenuProps` are not actually being spread/passed to the popper child below. Seems like we should do that here (like is done with some other components in this PR), to support any additional props beyond `sx`, which is already passed explicitly. 

Also, we should exclude any props which would be overwritten/ignored, like `modifiers` (unless you want to add logic to conditionally use the provided prop instead of the default behavior)